data/djangocon-2015/videos/beyond-the-basics-with-elasticsearch-by-honza-kral.json

```diff
@@ -12,12 +12,14 @@
   "videos": [
     {
       "url": "https://www.youtube.com/watch?v=RAJiFK1bHO0",
-      "type": "youtube",
-      "length": 0
+      "length": 0,
+      "type": "youtube"
     }
   ],
   "source_url": "https://www.youtube.com/watch?v=RAJiFK1bHO0",
   "tags": [],
-  "speakers": [],
+  "speakers": [
+    "Honza Kral"
```

Comment:
I noticed that Honza's name is spelled two ways across our data. This is the all-ascii way. I'll write up an issue to fix his name in the places where it's all-ascii.